projects/elife.yaml

```diff
@@ -2018,3 +2018,13 @@ fastly-logs:
             "{instance}-elife-fastly":
                 project: elife-fastly
     aws-alt: {}
+
+data-pipeline:
+    description: Data consolidation project
+    domain: false
+    intdomain: false
+    aws:
+        ec2: false
+        s3:
+            "{instance}-elife-data-pipeline":
```

Comment:
do we need a `deletion-policy`?

Reply:
`delete` (throw away with the project instance) or `retain` (preserve after deletion)?

Reply:
added